app/Services/Manager.php

```diff
@@ -148,6 +148,9 @@ public function getUserActivity($id, $model)
             $signup->reportback->updated_at = new Carbon($signup->reportback->updated_at);
             $signup->reportback->updated_at = $signup->reportback->updated_at->format('Y-m-d');
 
+            // Return set flagged status to 'pending' if it is false.
+            $signup->reportback->flagged = ($signup->reportback->flagged) ? $signup->reportback->flagged : 'pending';
```

Comment:
oh well i guess that's just how I always do ternary's, especially when im just testing if a variable isset, null, or false, since it reads more like a test to me. But if that's not standard I can remove them!